test/unit/package_test.rb

```diff
@@ -233,158 +233,158 @@ def test_centimetres_girth_cylinder
 
   end
 
-  def test_centimetres_girth
+  test "#centimetres_girth" do
     refute_predicate @package, :cylinder?
     assert_in_delta 22, @package.centimetres(:girth), 1
     assert_in_delta 22, @package.centimetres(:around), 1
     assert_in_delta 22, @package.centimetres(:circumference), 1
   end
 
-  def test_centimetres_volume_cylinder
+  test "#centimetres for the volume of acylinder" do
     @package = Package.new(@weight, @dimensions, @options.merge(cylinder: true, units: :metric, dim_units: :metric))
 
     assert_predicate @package, :cylinder?
     assert_in_delta 522.4, @package.centimetres(:volume), 1
   end
 
-  def test_centimetres_volume
+  test "#centimetres for volume" do
     refute_predicate @package, :cylinder?
     assert_equal 210, @package.centimetres(:volume)
   end
 
-  def test_centimetres_box_volume
+  test "#centimetres for box_volume" do
     assert_equal 210, @package.centimetres(:box_volume)
   end
 
-  def test_centimetres_unknown
+  test "#centimetres of an unknown value" do
     assert_nil @package.centimetres(:unknown)
   end
 
-  def test_centimetres_alias_cm
+  test "#centimetres alias to cm" do
     assert_equal @dimensions, @package.centimetres
     assert_equal @dimensions, @package.cm
   end
 
-  def test_weight
+  test "#weight" do
     assert_equal @mass, @package.weight
     assert_instance_of Measured::Weight, @package.weight
   end
 
-  def test_weight_actual
+  test "#weight actual" do
     assert_equal @mass, @package.weight(type: :actual)
     assert_instance_of Measured::Weight, @package.weight(type: :actual)
   end
 
-  def test_weight_volumetric
+  test "#weight volumetric" do
     assert_equal Measured::Weight.new(35, :grams), @package.weight(type: :volumetric)
   end
 
-  def test_weight_dimensional
+  test "#weight dimensional" do
     assert_equal Measured::Weight.new(35, :grams), @package.weight(type: :dimensional)
   end
 
-  def test_weight_billable_max_weight_and_volumetric
+  test "#weight billable is the max of weight and volumetric" do
     assert_equal Measured::Weight.new(100, :grams), @package.weight(type: :billable)
 
     @package = Package.new(500, [1, 1, 1], @options)
     assert_equal Measured::Weight.new(500, :grams), @package.weight(type: :billable)
   end
 
-  def test_grams_value
+  test "#grams value" do
     assert_equal 100, @package.grams
   end
 
-  def test_grams_accepts_options_with_type
+  test "#grams accepts options with type" do
     assert_in_delta 35, @package.grams(type: :volumetric), 1
   end
 
-  def test_grams_converts
+  test "#grams converts to another unit" do
```

Comment:
Maybe `#grams converts from another unit system`, since another unit would be something like `kg`?

Also, you could drop the `@package` initialization here and just use `@imperial_package`, I believe. I noticed the similar `#ounces` test below does that.